index.bs

```diff
@@ -282,6 +286,9 @@ spec: CORE-AAM; urlPrefix:https://www.w3.org/TR/core-aam-1.2
 spec: MEDIAQUERIES4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/
   type: dfn
     text: resolution media feature; url: #resolution
+spec:infra; urlPrefix: https://infra.spec.whatwg.org/
```

Comment:
I think we should not reference infra for the user agent since it has a different meaning there. Let's just say `user agent` without a link to the infra spec.

Reply:
Done!